fortio_main.go

```diff
@@ -229,34 +240,31 @@ func main() {
 	case "load":
 		fortioLoad(*curlFlag, percList)
 	case "redirect":
-		isServer = true
+		isServer = serverArgCheck()
```

Comment:
it's a bit hacky, basically to avoid 2 lines
```golang
isServer = true
serverArgCheck()
```
in 6 places (I suppose an alternative also ugly is make isServer global and set it in the function)